client/source/components/overview/sub-components/ZoomedPhoto.jsx

```diff
@@ -20,12 +17,49 @@ const ZoomedPhoto = function CreateZoomedPhotoComponent({
     return (() => { document.removeEventListener('keydown', handleKey); });
   }, []);
 
+  function imagePan(event) {
+    const image = event.target;
+    const mouseX = event.offsetX;
+    const mouseY = event.offsetY;
+    const imageW = image.offsetWidth;
+    const imageH = image.offsetHeight;
+    const xNew = (mouseX * 100) / imageW;
+    const yNew = (mouseY * 100) / imageH;
+    image.style.transformOrigin = `${xNew}% ${yNew}%`;
+  }
+
+  const handleClick = useCallback((event) => {
+    const elem = event.target.parentNode;
+    elem.removeEventListener('mousemove', imagePan);
+    setClicked((previous) => !previous);
+    if (!clicked) {
+      elem.addEventListener('mousemove', imagePan);
+    }
+  });
+
   return (
-    <Styles.ZoomedPhotoDiv color={color} onClick={handleClick}>
-      <Styles.PhotoTileInnerDiv>
-        <Styles.PhotoTileImage style={{ maxHeight: '75vh' }} alt="large format product photograph" src={photo.url} />
-      </Styles.PhotoTileInnerDiv>
-    </Styles.ZoomedPhotoDiv>
+    <div style={{ width: '60vw', height: '60vh', position: 'absolute' }}>
+      <Styles.OverlayDiv onClick={callback} />
+      <Styles.ZoomedPhotoDiv color={color}>
+        <Styles.PhotoTileInnerDiv>
+          <Styles.ZoomedPhotoTileImage
+            style={{
+              maxHeight: '75vh'
+            }}
+            clicked={clicked}
+            alt="large format product photograph"
+            src={photo.url}
+            onClick={handleClick}
+          />
+        </Styles.PhotoTileInnerDiv>
+      </Styles.ZoomedPhotoDiv>
+      {/* <div style={{ position: 'relative', zIndex: '102', backgroundColor: 'white' }}>
```

Comment:
Is this for later, or do you need to cut the comments?

Reply:
It is for later, as there are additional requirements not yet met. Technicially I should have written a comment just above, will fix.